### PR TITLE
Use LiveData-ktx 'map' & 'switechMap' extensions function replace Transformations function

### PR DESCRIPTION
**Pull request by madroidmaq**
Replace the `map` and `switchMap` static functions in the `Transformations` class with the `map` and `switchMap` extension functions in the `lifecycle-livedata-ktx` dependency package, which is more in line with the `Kotlin` syntax.

### CODE REVIEW

**Comment by madroidmaq** [comment]
I signed it!

**Comment by tiembo** [comment]
> Thank you for the PR! Added a minor nit.

Hmm, I'm not seeing the line comment.  The nit was sorting the dependencies in alphabetical order.

File: app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantDetailViewModel.kt [hunk 1]
```diff
@@ -16,10 +16,7 @@
 
 package com.google.samples.apps.sunflower.viewmodels
 
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.Transformations
-import androidx.lifecycle.ViewModel
-import androidx.lifecycle.viewModelScope
+import androidx.lifecycle.*
```
[the remark this answers is not in the document]
Comment (madroidmaq) [reply app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantDetailViewModel.kt]: The [commit](https://github.com/googlesamples/android-sunflower/pull/403/commits/d53ff5eadc9fb4900e6865eedd89f3d0d9af9e20) fix this.

**Comment by madroidmaq** [comment]
@tiembo I fix it in this [commit(d53ff5e)](https://github.com/googlesamples/android-sunflower/pull/403/commits/d53ff5eadc9fb4900e6865eedd89f3d0d9af9e20).

File: app/build.gradle [hunk 2]
```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation "androidx.core:core-ktx:$rootProject.ktxVersion"
     implementation "androidx.lifecycle:lifecycle-extensions:$rootProject.lifecycleVersion"
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$rootProject.lifecycleVersion"
+    implementation "androidx.lifecycle:lifecycle-livedata-ktx:$rootProject.lifecycleVersion"
```
Comment (tiembo) [inline app/build.gradle]: Can you move this line above line 56?  This way the dependencies will be in alphabetical order.

Comment (madroidmaq) [reply app/build.gradle]: I finished.

**Comment by tiembo** [comment]
Thank you!

**Pull request merged**